lib/rubocop/ast/node.rb

```diff
@@ -98,7 +95,7 @@ class Node < Parser::AST::Node # rubocop:disable Metrics/ClassLength
         kwrestarg: :argument,
         blockarg: :argument,
         forward_arg: :argument,
-        shardowarg: :argument,
+        shadowarg: :argument,
```

Comment:
Typo since this was added, tests caught it when I started using `each_descendant(:argument)`